### DISCUSSION TITLE
ARC installation errors

### DISCUSSION THREAD

**Discussion by D0ZT** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#153] [Q&A]
Hi, I have been trying to install ARC on mac using the command: pip install ARC-Alkali-Rydberg-Calculator

but have been getting the error message; ERROR: Could not build wheels for ARC-Alkali-Rydberg-Calculator, which is required to install pyproject.toml-based projects

If anyone has advice that would be greatly appreciated.

**Comment by nikolasibalic** [comment]
You probably use Mac with M2 chip, and we don't have pre-build binaries for that on PyPI.

This problem was discussed before. Please just install developer tools on Mac, and your computer will be able to build then locally binaries. Instructions are [here](https://github.com/nikolasibalic/ARC-Alkali-Rydberg-Calculator/discussions/101#discussioncomment-2615542) and [here](https://github.com/nikolasibalic/ARC-Alkali-Rydberg-Calculator/discussions/101#discussioncomment-2615676). Hope this helps and sorry for delay in response.
